Applications/IK/test/constraintTest_ikOnly.osim

```diff
@@ -5941,7 +5941,7 @@
 						<groups/>
 					</WrapObjectSet>
 				</Body>
-				<Body name="toes_l">
+				<Body name="toes_left">
 					<mass>       0.09750000 </mass>
```

Comment:
Why was this renamed? Not that the naming is bad, but the other bodies were not renamed for clarity, so I assume this was renamed for some other reason.